zoo-css-challenge/style.css

```diff
@@ -81,12 +87,13 @@ footer {
 #badge {
   height: 120px;
   width: 120px;
-  background-color: #bbb;
+  background-color: #FFFFFF;
   border-radius: 50%;
   display: inline-block;
-  position: fixed;
+  position: absolute;
   bottom: 20px;
   right: 20px;
+  transition-duration: 0.4s;
 
   display: flex;
   justify-content: center;
```

Comment:
You need to move the badge to the top right. At the moment it is 20px from the bottom, as set using the `bottom` property, but there is another CSS property called `top` which does the reverse. You should be able to fix this now.